isic/ingest/models/accession.py

```diff
@@ -683,6 +683,10 @@ def generate_thumbnail(self) -> None:
                 # Load the image so the stream can be closed
                 img.load()
 
+        # handle 16-bit grayscale images (RCM tiles) by rescaling to 8-bit
+        if img.mode == "I;16":
+            img = img.point(lambda p: p / 256).convert("L")
```

Comment:
This version touches every pixel with Python code. A 3x speedup can be achieved with:
 ```suggestion
            img = Image.fromarray(np.right_shift(np.asarray(image), 8).astype(np.uint8))
```

Note, this will definitely lose any extra metadata in the PIL object, as it's recreating it from pixel values, but that shouldn't matter, considering we want to strip the images anyway.

Reply:
Also need a:
```
import numpy as np
```
at the top, and to directly add `numpy` to the list of requirements (it looks like it's an indirect one now).